<launch>
    <param name="init_name" value="perception_camera"/>
    <param name="cam_name" value="cam_0"/>
    <param name="lidar_name" value="lidar_0"/>
    <param name="config_path" value="/home/zyc/work/superg_agv/src/perception/perception_camera/"/>
    <param name="image_obstacle_topic" value="/perception_camera/image_obstacle_info"/>
    <param name="lidar_topic" value="/drivers/velodyne/velodyne_points"/>
    <param name="pub_obstacle_info_topic" value="/perception/detection_camera"/>
    <param name="pub_rviz_bounding_box_topic" value="/perception_camera/rviz/make_bounding_box"/>
    <param name="pub_rviz_bounding_box_info_topic" value="/perception_camera/rviz/make_bounding_box_info"/>
    <param name="pub_rviz_split_pointcloud_with_camera_fov_topic" value="/perception_camera/rviz/split_pointcloud_with_camera_fov"/>
    <param name="pub_rviz_split_pointcloud_with_image_obstacle_info_topic" value="/perception_camera/rviz/split_pointcloud_with_image_obstacle_info"/>

    <param name="image_obstacle_publish_topic" value="/perception_camera/image_obstacle_info"/>
    <param name="pb_path" value="/home/zyc/work/superg_agv/src/perception/perception_camera/scripts/detection_models/ssd_inception_v2_no_tensorrt.pb"/>
    <param name="cam_cfg_path" value="/home/zyc/work/superg_agv/src/perception/perception_camera/scripts/config/cam_0.yaml"/>
    <param name="sub_image_topic" value="/usb_cam/image_raw"/>

    <node pkg = "perception_camera" type = "camera_obstacle_detection" name = "perception_camera" output = "screen"/>
    <node pkg = "perception_camera" type = "main.py" name = "perception_camera_image" output = "screen"/>
</launch>